.github/workflows/elixir.yml

```diff
@@ -1,14 +1,14 @@
 name: Elixir CI
 
-on: push
+on: [push, pull_request]
```

Comment:
A slightly off-topic suggestion: maybe rename this workflow file to describe what it does? Like `tests.yml` or something like that?

Reply:
:+1: